interactions/api/models/gw.py

```diff
@@ -60,7 +61,9 @@ class ApplicationCommandPermissions(ClientSerializerMixin):
     application_id: Snowflake = field(converter=Snowflake)
     guild_id: Snowflake = field(converter=Snowflake)
     id: Snowflake = field(converter=Snowflake)
-    # permissions: List[Permission] = field(converter=convert_list(Permission))  # TODO fix circular import
+    # from ...client.models.command import Permission
+
+    # permissions: List[Permission] = field(converter=convert_list(Permission))
```

Comment:
Is this supposed to be here?

Reply:
Yeah it was circular import and I attempted a fix lol